sciflo/utils/misc.py

```diff
@@ -692,7 +693,8 @@ def getUserInfo():
 
     userInfo = pwd.getpwuid(os.getuid())
     userName = userInfo[0]
-    homeDir = userInfo[5]
+    # homeDir = userInfo[5]
+    homeDir = tempfile.gettempdir()
```

Comment:
wrap this in a try/catch. Default to home directory, but fallback to tempdir if the makedirs fails